### PR TITLE
Bump Eventing to v1.0

### PR DESCRIPTION
**Pull request by pierDipi**
- Bump eventing to 1.0 in `project.yaml`
- Fix patches
- Run `make generated-files`

### CODE REVIEW

File: openshift-knative-operator/hack/update-manifests.sh [hunk 1]
```diff
@@ -36,11 +36,7 @@ function download {
     file="${files[$i]}.yaml"
     target_file="$target_dir/$index-$file"
 
-    if [[ "${component}" == "serving" ]]; then
-      url="https://github.com/knative/$component/releases/download/knative-$version/$file"
-    else
-      url="https://github.com/knative/$component/releases/download/$version/$file"
-    fi
+    url="https://github.com/knative/$component/releases/download/knative-$version/$file"
```
Comment (pierDipi) [inline openshift-knative-operator/hack/update-manifests.sh:39]: @skonto I removed this since eventing needs the `knative-` prefix too.

**Comment by pierDipi** [comment]
All green!

/cc @devguyio @skonto

**Comment by skonto** [comment]
Leaving to @devguyio for the final approval.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

YAAAY

**Pull request merged**